java/core/src/java/org/apache/orc/impl/DynamicByteArray.java

```diff
@@ -222,10 +222,10 @@ public void setText(Text result, int offset, int length) {
    * @param out the stream to write to
    * @param offset the first offset to write
    * @param length the number of bytes to write
-   * @throws IOException
+   * @throws IOException if an I/O error occurs
```

Comment:
@belugabehr . Could you revert this file change from this PR?
- First of all, for the line 227 and 228, this is irrelevant. They are only style changes.
- Second, do we need to add this additional comment, `if an I/O error occurs`? This is a Java default semantic. It doesn't add any new information and ORC users don't think this function throws IOException always.

Reply:
Sure.  Just trying to be thorough and document everything, but I agree that this change does not add any value for this particular PR.